lib/checkers/zookeeper_health_checker.rb

```diff
@@ -66,6 +73,21 @@ def check
 
   private
 
+  def active_health_check(check_details)
+    node_path = zk_client.create("/#{hostname}", ephemeral: true, sequential: true)
+    zk_client.delete(node_path)
+
+    true
+  rescue => e
+    logger.error { "Caught error during health check: #{e.inspect}\n#{e.backtrace.join("\n")}" }
+    false
+  end
+
+  def zk_client
+    @zk_client ||= ZK.new("#{zk_connection_settings[:host]}:#{zk_connection_settings[:port]}/whazzup",
```

Comment:
Will this client properly initialize a new session automatically if the connection is severed?